src/hooks/useCart/useCart.tsx

```diff
@@ -323,8 +324,19 @@ export const useCart = (
         setCheckoutResult(transactionResponse);
         setCartState("PURCHASED");
       } catch (e) {
+        if (e.message === ERROR_MESSAGE.DUPLICATE_POD_INPUT) {
```

Comment:
Hmm, will this case run currently?

Reply:
Currently this error message does not seem to exist on the backend haha, unless I missed it

Reply:
ya this doesnt, its an old error message ps! removed!